README.md

```diff
@@ -10,7 +10,9 @@ An implementation of Resources in Ember.JS without decorators.
 _This is a [V2-format Addon](https://github.com/emberjs/rfcs/pull/507) with V1 compatibility_
 
 - [Installation](#installation)
-- [Usage](#usage)
+- [Why](#what-is-a-resource)
```

Comment:
Somehow I don't think it's a good idea to mismatch the table of content label (why) and respective section header (what is a resource)

Reply:
updated!